src/pages/map.jsx

```diff
@@ -88,7 +91,7 @@ const MapPage = () => {
                                                     <td style={{border: '0px hidden', padding: '5px'}}>
                                                         <dt>{story.map.name}</dt>
                                                         <dt>{story.map.location}</dt>
-                                                        <dt>{story.map.industries.join(', ')}</dt>
+                                                        <dt>{(story.map.industries || story.metadata.industries).join(', ')}</dt>
```

Comment:
hrmmmm

Maybe throw in a || [] at the end, just incase that one is empty too?

```suggestion
                                                        <dt>{(story.map.industries || story.metadata.industries || []).join(', ')}</dt>
```